### DISCUSSION TITLE
unexpected network error has occurred.

### DISCUSSION THREAD

**Discussion by suwu150** [teambit/bit#7161] [Help]
unexpected network error has occurred.

unexpected network error has occurred.
server responded with: "scope "--". error: fatal: the server is busy by other exports. total exports (including yours) in the queue: 2,
the current export-id in queue is "1678862345977".
in a few minutes, bit.cloud will try to complete the export for you.
if the remote scopes are not updated, please contact support.
"

**Comment by itaymendel** [comment]
for issues with bit.cloud, please connect with support at support.bit.cloud
